modules/data-entry/src/main/frontend/src/questionnaire/Form.jsx

```diff
@@ -522,6 +519,7 @@ function Form (props) {
               paginationEnabled={paginationEnabled}
               questionnaireData={data.questionnaire}
               setPagesCallback={setPages}
+              savePage={handlePageChange}
```

Comment:
The prop name for event handlers should reflect **what event they handle** and **not how that event is handled**. What is being done externally when the event occurs in `FormPagination` is not `FormPagination`'s concern. Please rename `savePage` to `onPageChange`.
```suggestion
              onPageChange={handlePageChange}
```